core/src/main/java/io/questdb/cairo/GenericRecordMetadata.java

```diff
@@ -129,6 +129,15 @@ public GenericRecordMetadata add(int i, TableColumnMetadata meta) {
             columnCount++;
             return this;
         }
+        // Check if the older column with the same name was deleted.
```

Comment:
> this might be unnecessary and all metadata is compressed

So the attempt to differentiate metadata was reverted? Good to know. Will remove this code.

Reply:
yeah, aplogies!

Reply:
Removed in 0da96063674df9fb51bacc334a93b9d4971f1a16